src/main/java/cloud/fogbow/common/models/SystemUser.java

```diff
@@ -28,4 +36,25 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(identityProviderId, getId(), getName());
     }
+
+    public static String serialize(SystemUser systemUser) throws UnexpectedException {
```

Comment:
is this SerializedEntityHolder used by any other class than this SystemUser stuff? If not, I'd vote to delete it.

Reply:
Looking at the code, it seems that SerializedEntityHolder is important for the RecoveryService on RAS. So, I think we should not change the code.